lib/ssl/src/tls_connection.erl

```diff
@@ -741,24 +745,27 @@ hello(internal, #server_hello{extensions = Extensions} = Hello,
                  renegotiation = {Renegotiation, _},
                  ocsp_stapling_state = OcspState} = HsEnv,
 	     ssl_options = SslOptions} = State) ->
-    %% check if server has sent a certifucate status message in hello
-    IsCertStatusRequested = maps:is_key(ocsp_stapling, Extensions),
+    %% check if server has sent an empty certifucate status message in hello
+    IsCertStatusRequested = (undefined == maps:get(status_request, Extensions, false)),
+    #{ocsp_stapling := OcspStapling} = SslOptions,
     case tls_handshake:hello(Hello, SslOptions, ConnectionStates0, Renegotiation) of
 	#alert{} = Alert -> %%TODO
 	    ssl_connection:handle_own_alert(Alert, ReqVersion, hello,
                                             State#state{connection_env =
                                                             CEnv#connection_env{negotiated_version = ReqVersion},
                                                         handshake_env = HsEnv#handshake_env{
                                                             ocsp_stapling_state = OcspState#{
-                                                                is_server_hello_requested := IsCertStatusRequested}}
+                                                                ocsp_negotiated :=
```

Comment:
I think in this part we will not need to update the ocsp state as we are about to abort the handshake.

Reply:
done